fixtures/confusing-naming1.go

```diff
@@ -0,0 +1,47 @@
+// Test of confusing-naming rule.
+
+// Package pkg ...
+package pkg
+
+type foo struct{}
+
+func (t foo) aFoo() {
+	return
+}
+
+func (t *foo) AFoo() { // MATCH /Method 'AFoo' differs only by capitalization to method 'aFoo'/
+	return
+}
+
+type bar struct{}
+
+func (t *bar) aBar() {
+	return
+}
+
+func (t *bar) aFoo() { // Should not warn
+	return
+}
+
+func aGlobal() {
+
+}
+
+func AGlobal() { // MATCH /Method 'AGlobal' differs only by capitalization to method 'aGlobal'/
```

Comment:
Maybe we can show the file where `aGlobal` resides? For example, if `AGlobal` is in:

```
/foo/bar/baz/a.go
```

And `aglobal` is in:

```
/foo/bar/baz/qux/b.go
```
We can show:

```
Method 'AGlobal' differs only by capitalization to method 'aGlobal' in `qux/b.go`
```